### ISSUE TITLE
Test BasicKestrel_DotNet_ProductionScenario requires attention

### ISSUE THREAD

**Issue by DavidObando** [aspnet/Performance#112]
The test [`BasicKestrel_DotNet_ProductionScenario`](https://github.com/aspnet/Performance/blob/dev/test/Microsoft.AspNetCore.Tests.Performance/WebPerformance.cs#L73) deploys BasicKestrel and then sends a Ctrl+C in order to kill the process. It will then wait for up to 30 seconds and fail the test if the process hasn't exited after the time span.

Because of bug dotnet/cli#1327, this test has unrealistic expectations. The actual service process is a grand-child process of the one running the test, and the process chain will remain alive, so the test always fails. Until that CLI issue is fixed, I suggest this test gets decommissioned. Please let me know what you think.

/cc @SajayAntony @sivagms @NTaylorMullen

**Comment by DavidObando** [comment]
Perhaps the required change is simply to kill the process tree instead of sending Ctrl+C, and not asserting.

```C#
Assert.True(process.WaitForExit((int)TimeSpan.FromSeconds(30).TotalMilliseconds));
```

Becomes:
```C#
process.KillTree();
```

**Comment by DavidObando** [comment]
Addressed by #133
